src/main/java/hudson/remoting/RemoteInvocationHandler.java

```diff
@@ -851,7 +851,8 @@ private void onChannelTermination(Channel channel) {
      *
      * @see UserRPCRequest
      */
-    static class RPCRequest extends Request<Serializable,Throwable> implements DelegatingCallable<Serializable,Throwable> {
+    static class RPCRequest extends Request<Serializable,Throwable> implements DelegatingCallable<Serializable,Throwable>, InternalCallable<Serializable, Throwable> {
```

Comment:
```suggestion
    private static class RPCRequest extends Request<Serializable,Throwable> implements DelegatingCallable<Serializable,Throwable>, InternalCallable<Serializable, Throwable> {
```

Reply:
Wasn't me: `UserRequest` uses this in https://github.com/jenkinsci/remoting/blob/92ecf07540c9e2d8ef3c76f8008afcc8a160e505/src/main/java/hudson/remoting/UserRequest.java#L199